### ISSUE TITLE
If Kylie Jenner committed suicide....

### ISSUE THREAD

**Issue by cernhofer** [uchicago-computation-workshop/anna_mueller#24]
I wonder if the theories presented in this paper could be extended beyond one's local community in reaction to the suicide of a public figure. With the ever increasing popularity of social media and celebrity presence, I don't think social diffusion and cohesion necessarily depend on physical distance. Additionally, the 'social groups where our daily lives unfold' can be found on facebook/instagram/twitter/etc. where individuals can come together and define the social meaning of various world events regardless of geographical location. Could these online social groups serve as a proxy for one's local community, widening the set of individuals affected by a more public suicide?

Basically I want to know if my risk of suicide increases if my girl Kylie kills herself.

**Comment by bensoltoff** [comment]
@cernhofer she'll never do it, she has too much to live for.

Also, I just had to Google her to remember who she is. And she's way younger than me. Gross.

**Comment by cernhofer** [comment]
@bensoltoff Thank goodness- I don't think I could handle loosing that treasure. And yes she's a child but we still love her.
